### ISSUE TITLE
[BUG] menubar not rendered

### ISSUE THREAD

**Issue by R-chie** [compodoc/compodoc#1109]
- _version of compodoc, Node.js used_ 
compodoc: 1.1.14
node: 14.15.1
- tsconfig.json
```    
"compileOnSave": false,
  "compilerOptions": {
    "importHelpers": true,
    "module": "es2020",
    "outDir": "./dist/out-tsc",
    "sourceMap": true,
    "declaration": false,
    "moduleResolution": "node",
    "experimentalDecorators": true,
    "target": "es2015",
    "typeRoots": [
      "node_modules/@types"
    ],
    "lib": [
      "es2017",
      "dom"
    ],
    "plugins": [
      { "name": "tslint-language-service"}
    ]
  }
```
- for reproducing 
1. git clone https://github.com/akveo/ngx-admin.git
2. npm i
3. update compodoc to 1.1.14
4. npm run docs
![compodoc1](https://user-images.githubusercontent.com/9362723/127920913-ccb6177e-e7b6-444b-9a72-df49e68eb306.jpg)
5. edit index.html <script type='module' src="./js/menu-wc.js" defer></script> to <script src="./js/menu-wc.js" defer></script>
6. result 
![compodoc2](https://user-images.githubusercontent.com/9362723/127921129-dae2d83a-70da-4501-9465-c2b9e7ad8362.jpg)

**Comment by vogloblinsky** [comment]
Cannot reproduce the bug following the steps you provide.

**Comment by R-chie** [comment]
> 
> 
> Cannot reproduce the bug following the steps you provide.

Steps reproduced on 2 different Win 10 Pro PCs
Let me know for providing you any additional information

**Comment by ArsProgramma** [comment]
I can reproduce the bug mentioned as well, it even works with a brand new Angular application (v12).
**tl;dr: The clue seems to be to open the documentation locally, not served.**

While the served documentation (on http://127.0.0.1:8080/) is working absolutly fine,
opening the documentation locally isn't.

The only one problem is, that the compodoc-sidemenu is not rendered.
If, however, i follow the steps the OT describes the menu appears.

**Comment by vogloblinsky** [comment]
@R-chie is your issue with or without local server instance ?

**Comment by R-chie** [comment]
@vogloblinsky yes this issue without serve, if serve it works fine

**Comment by millbj92** [comment]
Experiencing the same issue. On FF and Chrome. Seems to be releated to CORS. I can host with a server all day, but at least for me, right now, this does not work when built to be static.

source: 
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSRequestNotHttp

**Comment by vogloblinsky** [comment]
The problem with the rendering of the Web Component of the menu is the compatibility with IE11.
The syntax you drop for fixing the ```file://```issue can add some strange results in IE11.
The combination of one script tag with `type="module"`and another without guarantee that in IE11, the file used for rendering the menu wil be the ES5 version, and not the ES6 one.
@millbj92 can you test your fix in IE11?

**Comment by millbj92** [comment]
I'm running Ubuntu 21.04 which unfortunately runs under Wayland. I can't get Wine. I'll boot up a windows pc a little later and check it out. It also noticed that every page changed caused a full age reload. I'm not sure if that's default for the static site or if I did that. I did see the fallback to es5 but for whatever reason when I commented out the es6 script the problem persisted.

**Comment by R-chie** [comment]
@vogloblinsky as like on issue description, i `edit index.html <script type='module' src="./js/menu-wc.js" defer></script> to <script src="./js/menu-wc.js" defer></script>` and checked  IE v11.789.19041.0, it's ok. It's same that @millbj92 fix

**Comment by millbj92** [comment]
Had a little more time for testing today, and dusting off my old windows machine. Works great with everything. Again, I'm a first time contributor so might want to stick this in a howfix branch first and check it out yourself.

**Comment by vogloblinsky** [comment]
Ok i will accept your PR @millbj92

**Comment by millbj92** [comment]
Thanks man you're a legend
